compute/cker/include/cker/operation/optimized/BinaryArithmeticOps.h

```diff
@@ -213,22 +353,46 @@ inline void BroadcastAddFivefold(const BinaryArithmeticOpParam &params,
   }
 }
 
+inline void BroadcastAddDispatchQuant8(const BinaryArithmeticOpParam &params,
+                                       const Shape &input1_shape, const uint8_t *input1_data,
+                                       const Shape &input2_shape, const uint8_t *input2_data,
+                                       const Shape &output_shape, uint8_t *output_data)
+{
+  if (params.broadcast_category == BroadcastableOpCategory::kGenericBroadcast)
+  {
+    const std::function<uint8_t(const BinaryArithmeticOpParam &, const uint8_t &, const uint8_t &)>
+        fn =
+            [](const BinaryArithmeticOpParam &params, const uint8_t &a, const uint8_t &b) -> float {
+      return static_cast<uint8_t>(quant8_sum(params, a, b));
+    };
+    reference::BroadcastBinaryArithmeticOpSlowQuant8(params, input1_shape, input1_data,
```

Comment:
if you modify this line to `return reference::BroadcastBinaryArithmeticOpSlowQuant8()`
then, you can remove "else" line.

Reply:
It wouldn't affect result or performance I guess. Please correct me if you feel otherwise.